### PR TITLE
Update Chinese language

### PR DESCRIPTION
**Pull request by awaysoft**

### CODE REVIEW

File: modules/system/lang/en/lang.php [hunk 1]
```diff
@@ -26,7 +26,7 @@
         'se' => 'Swedish',
         'sk' => 'Slovak (Slovakia)',
         'tr' => 'Turkish',
-        'zh-cn' => 'Chinese (China)',
+        'zh-cn' => '简体中文',
```
Comment (daftspunk) [inline modules/system/lang/en/lang.php]: This should be the word "Chinese" in English

Comment (awaysoft) [inline modules/system/lang/en/lang.php]: I think use native language will more friendly, maybe user does not know little English?

Comment (awaysoft) [inline modules/system/lang/en/lang.php]: And Chinese has two:Simplified Chinese and Traditional Chinese, I think 简体中文 and 繁体中文 is more clearly to users.
PS:Traditional Chinese is used in Taiwan,China.

Comment (daftspunk) [inline modules/system/lang/en/lang.php]: English (China) is fine. English should show English words.

Comment (awaysoft) [inline modules/system/lang/en/lang.php]: OK,I have pull a new request.

**Pull request closed without merging**